src/pudl/etl/__init__.py

```diff
@@ -31,7 +31,6 @@
 logger = pudl.logging_helpers.get_logger(__name__)
 
 default_assets = (
-    *load_assets_from_modules([output_assets], group_name="output_tables"),
```

Comment:
So we're just going to turn these off for now? I guess that seems reasonable until we have a real structure for managing them. Which will be soon!

Reply:
Yeah I figured we should turn them off for now because we don't have metadata for the example output assets. Still an open question as to where in our file structure we should be storing the assets.